File: backend/Security.Model/Application.cs
using System.Collections.Generic;

namespace Security.Model
{
    public class Application
    {
        /// <summary>
        /// Идентификатор приложения
        /// </summary>
        public int IdApplication { get; set; }

        /// <summary>
        /// Имя приложения. Является первичным ключов в БД
        /// </summary>
        public string AppName { get; set; }

        /// <summary>
        /// Описание
        /// </summary>
        public string Description { get; set; }
    }
}

File: frontend/SecurityManager/src/app/contracts/models/Application.ts
import { IEntity } from "./IEntity";

export class Application implements IEntity{
    
    public get Name() : string {
        return this.AppName;
    }

    public set Name(value: string){
        this.AppName = value;
    }
    
    get IdNumber(): number{
        return this.IdApplication;
    }
    
    IdApplication: number;
    AppName: string;
    Description: string;
}